frontend/src/pages/CalendarPage/index.js

```diff
@@ -101,6 +104,25 @@ class CalendarPage extends React.Component {
     });
   }
 
+  handleUpdateStatus = async (
+    values, newStatus
```

Comment:
Rename `values` to something else for more clarity since we clearly know an event object is being passed down. Tho you can probably just remove it altogether, per the comment above.